resources/popup.html

```diff
@@ -30,22 +31,33 @@
             <img id="statusImage" class="col s12" src="img/icon/off.png">
         </div>
 
-        <div class="row center">
-            <div class="col s4">
-                <a id="statusToggle" class="btn waves-effect lime waves-light">
+        <div class="row center" style="margin-bottom: 10px">
+            <div class="col s6">
+                <a id="statusToggle" class="btn waves-effect lime waves-light tooltipped"
+                   data-position="top" data-delay="500" data-tooltip="Power">
```

Comment:
You are right, I was doubting when i wrote this. But I could not think of a better (descriptive) name